src/node_i18n.cc

```diff
@@ -62,6 +68,80 @@ bool flag_icu_data_dir = false;
 
 namespace i18n {
 
+#define TYPE_ICU "icu"
+#define TYPE_UNICODE "unicode"
+#define TYPE_CLDR "cldr"
+#define TYPE_TZ "tz"
+// static const int TYPE_COUNT = 4;
+
+// static const char *VERSION_TYPES[TYPE_COUNT] = {
+//   TYPE_ICU,
+//   TYPE_UNICODE,
+//   TYPE_CLDR,
+//   TYPE_TZ
+// };
+
+
+/**
+ * This is the workhorse function that deals with the actual version info.
+ * Get an ICU version.
+ * @param type the type of version to get. One of VERSION_TYPES
+ * @param buf optional buffer for result
+ * @param status ICU error status. If failure, assume result is undefined.
+ * @return version number, or NULL. May or may not be buf.
+ */
+static const char *GetVersion(const char *type,
+char buf[U_MAX_VERSION_STRING_LENGTH],
```

Comment:
giving up on trying to use the ICU convention of "UErrorCode& status" here… I can't keep the NOLINT, arg indent, AND stay under the line width. What does cpplint have against non-const references?  :-1: